include/ipfixprobe/ipfix-elements.hpp

```diff
@@ -258,6 +258,21 @@ namespace ipxp {
 #define WG_SRC_PEER(F)                F(8057,    1101,   4,   nullptr)
 #define WG_DST_PEER(F)                F(8057,    1102,   4,   nullptr)
 
+#define NTS_MEAN(F)                   F(8057,    1020,  4,    nullptr)
+#define NTS_MIN(F)                    F(8057,    1021,  4,    nullptr)
+#define NTS_MAX(F)                    F(8057,    1022,  4,    nullptr)
```

Comment:
By the way, if min/max are packet sizes starting from L3, consider using standard IANA fields `minimumIpTotalLength`/`maximumIpTotalLength`. If min/max are packet size starting from L2 consider using standard IANA fields `minimumLayer2TotalLength`/`maximumLayer2TotalLength`.

See https://www.iana.org/assignments/ipfix/ipfix.xhtml

Reply:
It is the min/max value of the payload from the `pkt.payload_len_wire`. So, starting from L4.

Reply:
Ok, then there is not a standard field for it.